modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using an RPM package. 
```

Comment:
Can we add a small paragraph on the pre-reqs here? Those are:

- MicroShift installation >=V4.14 (RPM or ostree based)
- 250 MB of additional RAM recommended, though not requested

Reply:
Ill add a prereqs section, thank you!